solidity/contracts/middleware/InterchainQueryRouter.sol

```diff
@@ -38,7 +35,7 @@ contract InterchainQueryRouter is Router, IInterchainQueryRouter {
      * @param originDomain The domain of the chain to return the result to.
      * @param sender The address to receive the result.
      */
-    event QueryReturned(uint32 indexed originDomain, address indexed sender);
+    event QueryReturned(uint32 indexed originDomain, bytes32 indexed sender);
```

Comment:
Sorry I'm not sure I understand the difference between QueryReturned and QueryResolved? Returned to the origin chain vs resolved on the origin chain, what's the difference?

Reply:
I think maybe the natspec for QueryReturned is wrong?

Reply:
QueryExecuted might be a clearer name?

Reply:
> Sorry I'm not sure I understand the difference between QueryReturned and QueryResolved? Returned to the origin chain vs resolved on the origin chain, what's the difference?
are you arguing they should be the same event emitted on each chain?

Reply:
renamed to `QueryExecuted`